java/amazon-kinesis-producer/pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.amazonaws</groupId>
     <artifactId>amazon-kinesis-producer</artifactId>
-    <version>0.14.0</version>
+    <version>0.14.1-SNAPSHOT</version>
```

Comment:
Since we plan on rolling out a new version with this, let's update this to 0.14.1